BUILDING.md

```diff
@@ -22,7 +22,9 @@ Building requires access to an installation of Geode. There are two ways to achi
 * Pass in `GEODE_ROOT` during the CMake configuration step.
   * e.g.  add `-DGEODE_ROOT=/path/to/geode` to the _initial_ `cmake` execution command.
 
-
+## Setting Path to OpenSSL
```

Comment:
This should only be necessary if not using a an OpenSSL installed in a well known location or provided by the operating system.

Reply:
Thanks. Updated accordingly.